Library/Homebrew/keg_relocate.rb

```diff
@@ -319,6 +338,38 @@ def symlink_files
     symlink_files
   end
 
+  private
+
+  sig { returns(T::Boolean) }
+  def formula_disables_selective_replacement?
+    return false unless respond_to?(:name)
```

Comment:
```suggestion
```

this shouldn't be needed, if Sorbet is causing problems handle it using typical Sorbet style